Utilities/date_time.h

```diff
@@ -32,4 +32,12 @@ namespace date_time
 		strftime(str, sizeof(str), "%Y%m%d%H%M%S", &now);
 		return str;
 	}
+
+	static inline std::string current_time_with_underscores()
```

Comment:
Make it a single function using template.
eg. `template<bool use_underscores = false> current_time()`
`use_underscores ? :  "%Y_%m_%d_%H_%M_%S" : "%Y%m%d%H%M%S"`

Reply:
I went a step further and made it a choice for the separator.